apps/api/src/expenses/expenses.controller.spec.ts

```diff
@@ -97,7 +102,7 @@ describe('ExpensesController', () => {
 
       const createDto: CreateExpenseDto = { ...expense }
 
-      await expect(controller.create(createDto)).rejects.toThrow()
+      // await expect(controller.create(createDto, [])).rejects.toThrow()
```

Comment:
if this is intentionally commented out - please add why it should stay so

Reply:
OK. We don't validate the balance at the moment.
May be in the future if we need it - we can add it.